testcases/OpTestXive.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python2
+# IBM_PROLOG_BEGIN_TAG
+# This is an automatically generated prolog.
+#
+# $Source: op-test-framework/testcases/OpTestXive.py $
+#
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2017
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# IBM_PROLOG_END_TAG
+
+'''
+OpTestXive
+------------
+
+This module contain testcases related to XIVE.
+
+1.check for there are no xive related
+errors in dmesg log
+2.Enable kernel logging, basically generate traffic
+3.Enable console traffic by printing the 
+processes/tasks to the console
+4.check Xive is configured properly or not
+5.check system rebbot with xive=on
+6.check system reboot with xive=off
+7.validate interrupts handled by CPU's
+'''
+import time
+import subprocess
+import commands
+import re
+import sys
+import pexpect
+import os
+import time
+
+
+from common.OpTestConstants import OpTestConstants as BMC_CONST
+from common.OpTestError import OpTestError
+from common import OpTestHMC
+
+import unittest
```

Comment:
it can be moved up, 
1. python built-in modules
2. framework modules

Reply:
sure